data/human/hails.txt

```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
 		"neodymium and neodymium" "neodymium"
 		"neodymium or neodymium" "neodymium"
 
+phrase "friendly civilian"
+	word
+		"Be careful of"
+		"A warning about"
+		"Be suspicious of"
+		"Don't trust"
+	word
+		" "
+	word
+		"those space casinos. I blew"
+	word
+		" "
+	word
+		"a"
+		"two"
+		"three"
+		"four"
+		"five"
+	word
+		" "
+	word
+		"million credits on "
+	word
+		"Bourne"
+		"Reunion"
+		"Rust"
+		"Quicksilver"
+		"Shangri-La"
+		"Nimbus"
+		"Zug"
+		"Valhalla"
+		"Longjump"
+		"Glaze"
+		"Hestia"
+		"Earth"
+		"Winter"
+		"Sunracer"
+		"Twinstar"
+		"Farpoint"
+		"Tinker"
+		"Hephaestus"
+	word
+		" "
+	word
+		"blackjack."
+		"poker."
+		"backgammon."
+		"baccarat."
+		"roulette."
+
+phrase "friendly civilian"
+	word
+		"Have you read the latest edition of"
+	word
+		" "
+	word
+		"Cosmic Chronicles"
+		"Stellar Spectacles"
+		"Galactic Gazette"
+		"Quantum Quandaries"
+		"Nebula News"
+		"Phantom Planets"
+		"Vague Voyages"
+		"Dark Dimensions"
+		"Mysterious Moons"
+		"Silent Satellites"
+		"Zesty Zeniths"
+		"Daring Discoveries"
+		"Fiery Frontiers"
+		"Global Galaxies"
+		"Rapid Rockets"
+		"Orbital Overload"
+		"Stars and Spaceships"
+	word
+		"? It says that"
+	word
+		" "
+	word
+		"pulse showers are good for your quantum pores."
+		"exposing yourself to neodymium increases your risk of spacepox."
+		"shoddily grown ion algae exhausts your psychic karma."
+		"laser acupuncture lets you communicate with nebula ghosts."
+		"star crystals bring luck to long distance relationships."
+		"consuming moon dust accelerates hair growth but causes lunacy."
+		"excessive exposure to zero gravity can lead to phantom limb syndrome."
+		"alien prosthetics often come with unexpected telepathic abilities."
+		"stardate tattoos increase your chances of alien abduction."
+
+phrase "friendly civilian"
+	word
+		"I'm no weapons expert, but it feels better firing"
+	word
+		" "
+	word
+		"energy blasters"
+		"particle cannons"
+		"proton guns"
+	word
+		" "
+	word
+		"than"
+		"compared to"
+	word
+		" "
+	word
+		"beam lasers."
+		"heavy lasers."
+
+phrase "friendly civilian"
+	word
+		"I shouldn't have sold my old ship. It was a"
+	word
+		" "
+	word
+		"Shuttle"
+		"Star Barge"
+		"Sparrow"
+		"Heavy Shuttle"
+		"Freighter"
+		"Fury"
+	word
+		" "
+	word
+		"that never had"
```

Comment:
Doesn't really fit. It almost sounds like the issues in the next few lines are things you _want_ your ship to have. I'd rephrase it, or maybe change `It was a"` -> `It was just a"`, and `"that never had"` -> `"but it never had"`